admin/config-ui/fields/class-field-separator.php

```diff
@@ -21,6 +21,7 @@ public function __construct() {
 
 		$this->add_choice( 'sc-dash', '-', __( 'Dash', 'wordpress-seo' ) );
 		$this->add_choice( 'sc-ndash', '&ndash;', __( 'En dash', 'wordpress-seo' ) );
+		$this->add_choice( 'sc-colon', ':', __( 'Colon', 'wordpress-seo' ) );
```

Comment:
You might consider using `&colon;` instead of `:`

Reply:
For consistency with search appearance, let's use `:` here as well.